package.json

```diff
@@ -14,16 +14,18 @@
   "dependencies": {
     "autoprefixer": "^10.4.12",
     "postcss": "^8.4.17",
+    "prop-types": "^15.8.1",
```

Comment:
Pourquoi ?

Reply:
Added to devDependencies (react-simple-maps doesn't work without it)